STM32Cube/Tasks/state_estimation.h

```diff
@@ -8,6 +8,10 @@
 #ifndef STATE_ESTIMATION_H_
 #define STATE_ESTIMATION_H_
 
+#include "stm32h7xx_hal.h"
 
+extern UART_HandleTypeDef huart4;
```

Comment:
this should not exist since we have printf_ now

Reply:
removed